README.md

```diff
@@ -3,18 +3,18 @@
 
 # Source Code Annotation Tool
 
-Training Machine Learning models often requires large datasets to be duly annotated.
+Training Machine Learning models often require large datasets to be duly annotated.
```

Comment:
I believe it's _requires_.

_Training Machine Learning models_ often requires
_It_ often requires